app/src/main/java/com/geekskool/dhobi/Models/Student.java

```diff
@@ -0,0 +1,94 @@
+package com.geekskool.dhobi.Models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by manisharana on 16/2/17.
+ */
+public class Student implements Serializable {
+
+    private String name;
+    private boolean isPreviousStudent;
+    private String gender;   // convert to enum
+    private int age;
+    private int deposit;
+    private ArrayList<Purchase> purchases;
+    private ArrayList<Laundry> laundryList;
```

Comment:
Rather than separate deposit, purchases, and laundry into 3 parallel types of data, they could be captured in a field which effectively represents a table (internally -- this table wouldn't be displayed to users).

For example:

`Finances`, a custom collection which contains an `ArrayList` internally of `Deposit`, `Purchase`, and `Laundry`. These 3 could potentially (preferably) adhere to the same interface... though preferably they would *not* share a super class.